invenio_jobs/resources/config.py

```diff
@@ -37,15 +38,40 @@ class JobsResourceConfig(ResourceConfig, ConfiguratorMixin):
     url_prefix = "/jobs"
     routes = {
         "list": "",
-        "item": "/<id>",
+        "item": "/<job_id>",
     }
 
     # Request parsing
     request_read_args = {}
-    request_view_args = {"id": ma.fields.Int()}
+    request_view_args = {"job_id": ma.fields.Int()}
     request_search_args = JobsSearchRequestArgsSchema
 
     error_handlers = {
         **ErrorHandlersMixin.error_handlers,
         # TODO: Add custom error handlers here
     }
+
+
+class RunsResourceConfig(ResourceConfig, ConfiguratorMixin):
+    """Runs resource config."""
+
+    # Blueprint configuration
+    blueprint_name = "job_runs"
+    url_prefix = ""
+
+    routes = {
+        "runs": "/jobs/<job_id>/runs",
+        "run_item": "/jobs/<job_id>/runs/<run_id>",
+        "logs": "/jobs/<job_id>/runs/<run_id>/logs",
+    }
+
+    # Request parsing
+    request_view_args = {
+        "job_id": ma.fields.Int(),
+        "run_id": ma.fields.Int(),
```

Comment:
```suggestion
        "job_id": ma.fields.UUID(),
        "run_id": ma.fields.UUID(),
```

minor: to match the datamodel

Reply:
Fixed.